lib/streamlit/config.py

```diff
@@ -468,8 +468,8 @@ def _logger_message_format() -> str:
         traceback show in the browser also, but they will be removed in the
         future.
```

Comment:
There should be a first updated version of the doc string to reflect the new type and values (does not have to be super polished which Debbie will help with)

Reply:
It looks like my edit got force merged out, but I think  Laura is going to cherry pick it back in.